e2e/protractor/protractor.excludes.json

```diff
@@ -21,31 +21,5 @@
   "C307106" : "date test fail migrate to playwright https://alfresco.atlassian.net/browse/ACS-6185 ",
   "C269003" : "date test fail migrate to playwright https://alfresco.atlassian.net/browse/ACS-6185 ",
   "C290018" : "https://alfresco.atlassian.net/browse/ACA-4731",
-  "C277727" : "https://alfresco.atlassian.net/browse/ACS-6672",
-  "C277728" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C277740" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C277738" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C277734" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C277736" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C277732" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C299189" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C299195" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "MNT-21058" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C286384" : "https://alfresco.atlassian.net/browse/ACS-6688",
-  "C286379" : "https://alfresco.atlassian.net/browse/ACS-6688",
```

Comment:
I can see that some of these tests like this one are also in playwright probably (for example please notice that this ID is also removed from excluded in e2e/playwright/viewer/exclude.tests.json file in that PR. 

Are we running the same tests twice - one for playwright and one for protractor?

I guess it should not be a case because we should not run protractor tests anymore after migration to save time and resources.

Reply:
okay , I removed few protractor test that are got duplicated after remove of excluded test.